src/components/DayCard.spec.js

```diff
@@ -0,0 +1,14 @@
+import DayCard from './DayCard.js';
+import { render, screen } from '@testing-library/react';
+
+describe('DayCard', () => {
+  it('renders an article with a header and a paragrpah', () => {
```

Comment:
Small typo
```suggestion
  it('renders an article with a header and a paragraph', () => {
```

Reply:
As you are rendering a heading and a span element, maybe it is better so write her something like: "renders the day and date of a trip"

Reply:
uups, that's true a don't even render a paragraph 😂